verifier/verifier.go

```diff
@@ -198,24 +217,27 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 		for _, pc := range pluginCapabilities {
 			// skip the revocation capability if the trust policy is configured to skip it
 			if outcome.VerificationLevel.Enforcement[trustpolicy.TypeRevocation] == trustpolicy.ActionSkip && pc == proto.CapabilityRevocationCheckVerifier {
+				logger.Debugf("Skip the verification capability %v because of verification action setting as skipped", pc)
 				continue
 			}
 			capabilitiesToVerify = append(capabilitiesToVerify, pc)
 		}
 
 		if len(capabilitiesToVerify) > 0 {
+			logger.Debugf("Execute verification plugin with capabilities %v", capabilitiesToVerify)
 			response, err := executePlugin(ctx, installedPlugin, trustPolicy, capabilitiesToVerify, outcome.EnvelopeContent, pluginConfig)
 			if err != nil {
 				return err
 			}
-			return processPluginResponse(capabilitiesToVerify, response, outcome)
+
+			logger.Debugf("Plugin VerifySignatureResponse: %+v", *response)
```

Comment:
Any idea why we need to dereference?

Reply:
Also should we move this to pluginManager or the module that's actually unmarshalling response ?

Reply:
adding a `*` can remove the `&` symbol in the output to beautify the log.
However, moved this line to plugin package and removed the `*`.